.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       export CONFIG=linux64
       export DOCKER_IMAGE=quay.io/condaforge/linux-anvil-cos7-x86_64
       export AZURE=True
-      .scripts/run_docker_build.sh
+      .build_scripts/run_docker_build.sh
```

Comment:
Can you keep this `.scripts` to match the feedstocks?

Reply:
Yeah that makes sense. Plus we would need to update the docs if we changed this in the feedstocks

Also moved the creation scripts into `.github` instead of having a new directory. Maybe that is a bit clearer